studio/app/common/core/storage/s3_storage_controller.py

```diff
@@ -51,114 +46,50 @@ def make_experiment_remote_path(self, workspace_id: str, unique_id: str) -> str:
 
     def download_all_experiments_metas(self) -> bool:
         """
-        NOTE:
-          - この処理（config yaml files の S3 からのダウンロード）では、
-            ダウンロード対象ファイルリストの取得に、python module (boto3) ではなく、
-            外部コマンド (aws cli) を利用している
-          - aws cli を利用する事由
-            1. boto3 では、取得対象のファイルリストの Server(AWS) Side でのfilterをサポートしていない（2024.7時点）
-                - 「Prefix配下のファイルリストをすべて取得 → Client Side でのFilter」の操作手順となる
-            2. また 1. の操作を行う場合、Pagination の考慮も必要となる
-          - 上記のため、ファイルリストの取得には、aws cli を利用する形式としている
-            - aws cli (`aws s3 ls`) も、基本的には Server(AWS) Side のファイルリストfilterには非対応だが、
-              `aws s3 sync` の利用により、間接的に Server(AWS) Side での filterが利用可能
-            - なお aws cli の利用は暫定的な対応であり、最終的には boto3 でfilterを実現できることが望ましい
+        Download all experiment metadata files from S3 using boto3.

```

Comment:
For revisions that differ from the PR in content, please create a separate PR.
*For now, let's rename this PR, shall we?

Reply:
Sorry,  I thought I could add here because the description "fix env setting for remote_storage_controller" was close to the changes i made to use boto3. However, if you don't want to go with this approach, we can delete this last PR.